cmd/lakectl/cmd/refs.go

```diff
@@ -27,12 +25,10 @@ This command is expected to run on a bare repository (i.e. one created with 'lak
 Since a bare repo is expected, in case of transient failure, delete the repository and recreate it as bare and retry.`,
 	Example: "aws s3 cp s3://bucket/_lakefs/refs_manifest.json - | lakectl refs-load lakefs://my-bare-repository --manifest -",
 	Hidden:  true,
-	Args: cmdutils.ValidationChain(
-		cobra.ExactArgs(1),
-		cmdutils.FuncValidator(0, uri.ValidateRepoURI),
-	),
+	Args:    cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		repoURI := uri.Must(uri.Parse(args[0]))
+		repoURI := MustParseRepoURI("repository", args[0])
```

Comment:
Why do you need to pass the "repository" arg? Isn't it clear from the func call?

Reply:
What thinking about it as it is possible to use for copy repository or other future commands where you parse more than one repository name.
Also aligned with the other parse functions.